spark-cassandra-connector/src/main/scala/com/datastax/spark/connector/rdd/partitioner/CassandraPartitionGenerator.scala

```diff
@@ -79,12 +79,18 @@ private[connector] class CassandraPartitionGenerator[V, T <: Token[V]](
     val tokenRanges : Seq[TokenRange] = if (partitionMap != null) {
       // Compute map from (start) token to corresponding hosts based on the partition map.
       // Converting token map into Sequence of TokenRanges describing the ring for this table.
-      partitionMap.values.toSeq.map { partition =>
+      var ranges = partitionMap.values.toSeq.map { partition =>
```

Comment:
for readability: what is the type of ranges?

Reply:
The type would be Seq[TokenRange]